example/main.go

```diff
@@ -2,59 +2,27 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
-	"log"
 	"os"
 	"strings"
 
 	"github.com/wapc/wapc-go"
-	"github.com/wapc/wapc-go/engines/wasmtime"
+	"github.com/wapc/wapc-go/engines/wazero"
 )
 
-type Settings struct {
```

Comment:
The settings here are interesting, but they drift vs README and also clutter output. If we want to keep different examples from the README, we might consider a different source file.